model/Core/Classes/NamespaceMap.md

```diff
@@ -14,40 +14,25 @@ of ElementIDs to be used in SPDX content serialization in order to provide a mor
 human-readable and smaller serialized representation of the Elements.
 
 The content of a namespace map are suggestions for consistency in serialization but are not normatively enforced.
-Any serialization of the collection content SHOULD (rather than MUST) utilize the namespace map content.
+When a serialization only contains the elements in "X", the collection content SHOULD (rather than MUST) utilize
+the namespace map.
 A serialization MAY choose to use the namespace map content.
 A serialization MAY choose to use prefixes and namespaces other than the namespace map content.
 A serialization MAY choose to use no prefixes at all and rather use the more verbose full ElementID IRIs.
 
-If utilized the namespace map set of prefixes and namespaces MUST be implemented in a given serialization 
-form (e.g., json-ld or xml) as specified in the binding rules specification for that serialization and 
-utilizing the appropriate inherent or custom specified mechanism for that serialization.
-The namespace map itself is also conveyed as native SPDX content to support clarity, transparency and 
-consistency independent of any particular serialization form.
-
-A given serialization payload (whether file or streaming) MUST NOT contain multiple namespace maps with conflicting mappings.
-Any serialization of the collection content SHOULD (rather than MUST) utilize the namespace map content unless the single 
-serialization payload is intended to contain multiple peer collections each with their own namespace map.
-In this case the serializer MUST do one of the following:
-1) Diverge from the exact provided namespace mappings and utilize deconflicted mappings where appropriate.
-   This could be fully new prefix mappings as if the suggested namespace maps had not been provided.
-3) Use full ElementID IRIs instead of prefixed ElementID IRIs.
-
+If a serialization format supports prefixes or namespaces (e.g., JSON-LD context or XML XSD namespace), the namespace map
+must be represented in that format "native" to the serialization.
+The NamespaceMap itself will never be serialized as part of SPDX data if the serialization format support namespaces or prefixes.
```

Comment:
That is not strictly true, as it is just a SPDX element which could be serialized. But it has no semantic relation to the file which contains it.

Reply:
I'll remove this statement

Reply:
After reading David's comment below, it sounds like there are different views on whether this should be allowed to be serialized.

When the proposal was described, I thought it would never be included in an actual serialization.

Reply:
> When the proposal was described, I thought it would never be included in an actual serialization.

I think it is slightly different.

If I send such a `X` to the serializer. I Expect it to serialize all Elements that are contained while following the namespace map. And therefore it would not serialize the `X` itself. 

*But:* The `X` might contain another instance of `X` which would be serialized. There would then be no implication from the second `X` to the file it contains.

Reply:
This statement was removed